src/main/java/com/minecolonies/coremod/event/DebugRendererChunkBorder.java

```diff
@@ -87,7 +100,12 @@ public static void renderWorldLastEvent(@NotNull final RenderWorldLastEvent even
         stack.push();
         final Matrix4f matrix = stack.getLast().getMatrix();
 
-        for (final Map.Entry<Tuple<Integer, Integer>, Integer> c : colonies.entrySet())
+        // check control key pressed
+        boolean debugRenderChunkLoads = true || InputMappings.isKeyDown(Minecraft.getInstance().getMainWindow().getHandle(), 341);
```

Comment:
isn't the default false?

Reply:
oh thats a testing thing I forgot to remove^^ good catch

Reply:
would've been always on xD